docs/dictionary/function/byteOffset.lcdoc

```diff
@@ -2,11 +2,11 @@ Name: byteOffset
 
 Type: function
 
-Syntax: byteOffset(bytesToFind,binaryStringToSearch[,bytesToSkip])
+Syntax: byteOffset(<bytesToFind>,<binaryStringToSearch> [,bytesToSkip])
 
 Summary:
-Returns the number of bytes between the first byte of a value and an
-occurrence of a specified binaryString.
+<return|Returns> the number of <bytes> between the first <byte> of a <value> 
+and an occurrence of a specified binaryString.
```

Comment:
I suggest `... the first occurrence of a specified string of <binary data>` or something like that.